src/events/eventKeyboard.lua

```diff
@@ -11,5 +11,7 @@ function eventKeyboard(name, key, down, x, y)
         displayProfile(Player.players[name], name)
     elseif key == keys.LETTER_L then
         leaderboard.displayLeaderboard("global", 1, name)
+    elseif key == keys.LETTER_O or key == keys.LETTER_U then
```

Comment:
I haven't defined `keys.LETTER_O` and `keys.LETTER_U` in init.lua
https://github.com/Seniru/pewpew/blob/1e0deacfa4382451cfde47c57c4ed18a544e795c/src/init.lua#L105-L113

So you will have to define that. Binding the key codes are done using the `keys` table so you don't need to worry about it :+1: